docs/examples/userguide/extension_types/shrubbery.pyx

```diff
@@ -1,8 +1,10 @@
 from __future__ import print_function
 
+
 cdef class Shrubbery:
     cdef int width, height
```

Comment:
In pure python version the values ​​are on different lines, it is better to align them by compressing there or splitting here

Reply:
done thank you!